tasks/configure_storage_paths.yml

```
@@ -20,6 +20,23 @@
       else
       __mssql_storage_mode_default }}"
 
+- name: Append facts for the selinux role
+  vars:
+    selinux_fcontext:
+      - {
```

```suggestion
      - target: "{{ __mssql_storage_path }}"
        setype: "{{ 'mssql_db_t' if __mssql_storage_setting == 'defaultlogdir'
          else 'mssql_var_t' }}"
        ...
```
not sure why all of the selinux role examples mix json style `{ item: "value", ... }` with yaml style - should fix that in the selinux role to use only yaml style

---

I was thinking that this is a dictionary inside a list. I'll test it as you suggest.

---

It worked!